README.md

```diff
@@ -9,6 +9,11 @@ The Dotnet Core Conf CNB completes the setup of a Dotnet Core application built
 using a sequence of CNBs. As such, it will be the only non-optional CNB in that
 sequence and is not explicitly required by any CNB that precedes it.
 
+Downstream buildpacks can require the `dotnet-core-conf` dependency, however this
```

Comment:
Might be worth leveraging [this response](https://github.com/paketo-buildpacks/php-web/pull/13#issuecomment-615284511) to rework this section a bit.